src/js/DOMTabela.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+const domTabela = document.getElementById('domTabela')
+const redovi = document.getElementById('redovi')
+const kolone = document.getElementById('kolone')
+const dugme = document.getElementById('dugme')
+
+for (let i = 0; i < 21 + 1; i++) {
```

Comment:
`i < 21 + 1` je isto što i `i <= 21`

Reply:
Veruj mi da ni sam ne znam sta sam hteo sa ovim 21+1 ..